dll/win32/uxtheme/nonclient.c

```diff
@@ -270,6 +262,16 @@ void ThemeCalculateCaptionButtonsPos(HWND hWnd, HTHEME htheme)
     }
 }
 
+void ThemeCalculateCaptionButtonsPos(HWND hWnd, HTHEME htheme)
+{
+    WINDOWINFO wi = {sizeof(wi)};
+    if(!GetWindowInfo(hWnd, &wi))
```

Comment:
```suggestion
    INT btnHeight;
    WINDOWINFO wi = {sizeof(wi)};

    if (!GetWindowInfo(hWnd, &wi))
```

Reply:
P.S. review comment got resolved, but the code suggestion not applied.